unfetter-discover/unfetter-ui
# Issue 1142/percent complete

Fixes unfetter-discover/unfetter#1142

Percent complete of a baseline now based on average of completion for protect, detect, and respond weightings for all attack patterns and capabilities in the baseline.

### To test:
1. Pull this branch
2. Create a new baseline with only default attack patterns and their weightings
3. Save the baseline and there should be no specification of percent complete in the info bar at the top since all questions are provided by default
4. Edit the baseline and add in some attack patterns
5. Verify that the protect, detect, and respond answers are empty (i.e. unanswered)
6. Save the baseline without answering
7. Percent complete message and `Complete` button should be present in the info bar.  Percent complete should be average of the protect, detect, and respond percentages in the summary (below the heatmap).
